.circleci/config.yml

```diff
@@ -46,8 +47,30 @@ jobs:
               docker push "${DOCKER_REGISTRY}/${DOCKER_NAMESPACE}/${CIRCLE_PROJECT_REPONAME}:latest"
           name: "Build and Deploy Website Docker Image"
     working_directory: ~/repo
+  build_package:
+    docker:
+      - image: ubuntu:xenial
+    working_directory: /home/DEV.T1.TBS-SCT.GC.CA/dsamojle-ps
+    steps:
+      - checkout
+      - run:
+          command: |
+              apt-get update
+              apt-get install -y python3 python3-venv python3-dev build-essential
+              python3 -m venv .venv
+              . .venv/bin/activate
+              pip install wheel
+              pip install -r requirements.txt
+              tar -czvf track-web.tar.gz .venv track
+      - store_artifacts:
+          path: /home/DEV.T1.TBS-SCT.GC.CA/dsamojle-ps/track-web.tar.gz
+          destination: track-web.tar.gz
+          
 workflows:
   version: 2
   tracker:
     jobs:
-      - track_web 
+      - track_web
+      - build_package:
```

Comment:
Can you put a branch filter on this so it only builds master?

Reply:
ah yes, meant to do that